common/app/views/fragments/analytics.scala.html

```diff
@@ -69,20 +69,21 @@
 
         @* Limit to "modern browsers" *@
         if (guardian.isModernBrowser) {
+            try {
+                @* sessionStorage is part of the isModernBrowser test *@
+                var session = window.sessionStorage;
 
-            @* sessionStorage is part of the isModernBrowser test *@
-            var session = window.sessionStorage;
-
-            @*
-                If this key is still present then the user went to a new page before Omniture ran
-                and we did not count the page view
-            *@
-            if (session.getItem('gu-bounce-test') === 'true') {
-                var img = new Image();
-                img.src = '@{Configuration.debug.beaconUrl}/count/user-navigated-early.gif';
-            }
+                @*
+                    If this key is still present then the user went to a new page before Omniture ran
+                    and we did not count the page view
+                *@
+                if (session.getItem('gu-bounce-test') === 'true') {
+                    var img = new Image();
+                    img.src = '@{Configuration.debug.beaconUrl}/count/user-navigated-early.gif';
+                }
 
-            session.setItem('gu-bounce-test', 'true');
+                session.setItem('gu-bounce-test', 'true');
+            } catch (e) {};
```

Comment:
true and the bad stuff is our stats are messed up so we should probably take care here to avoid fooling us